README.md

```diff
@@ -125,6 +166,23 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
+1. lamp = Lamp.create(wishes_remaining: 3)
+   genie = Genie.create(name: "Genie")
+
+2. class Lamp < ActiveRecord::Base
+    has_many :genies
+   end
+
+   class Genie < ActiveRecord::Base
+     belongs_to :lamp
+   end
+
+3. lamp.update(wishes_remaining: 1)
+
+4. evil_lamp = Lamp.create(wishes_remaining: 3)
+   Genie.create(name: "Jafar", lamp: evil_lamp)
+
+5.
```

Comment:
You got most of everything, I think all we have to do now is set `genie.lamp = nil`